.github/workflows/slow.yaml

```diff
@@ -49,13 +49,20 @@ jobs:
 
     name: linux-distros(${{ matrix.os }},${{ matrix.compiler.CC }},${{ matrix.layer.nick }})
     env:
-      CC: ${{ matrix.compiler.CC }}
-      CXX: ${{ matrix.compiler.CXX }}
+      CC: /usr/bin/ccache ${{ matrix.compiler.CC }}
+      CXX: /usr/bin/ccache ${{ matrix.compiler.CXX }}
 
     steps:
       - name: Checkout Sources
         uses: actions/checkout@v4
 
+      - name: Setup ccache
+        uses: squid-cache/ccache-action@v1.2.14
+        with:
+          verbose: 2 # default 0
+          append-timestamp: false
```

Comment:
Do we have to make append-timestamp false? I could not tell from ccache-action "foo is foo" documentation; ccache-action [examples](https://github.com/hendrikmuhs/ccache-action) do not use this customization, and presumably work even when append-timestamp is true (by default), so I wonder whether/why we need this line.

Reply:
In my understanding, we do or we will not be able to reuse ccache across builds. I can experiment tho

Reply:
> In my understanding, we do or we will not be able to reuse ccache across builds.

... because ccache-action timestamps change slowly enough to stay the same within one workflow build/execution, even if individual workflow jobs could run minutes apart from each other? That would be a really weird/dangerous design!

> I can experiment tho

Thank you. I think such an experiment is worth it if it may result in keeping this poorly understood customization away from our workflows.

Reply:
Apparently ccache-action enumerates caches, so it can pick one and resstore it; I suppose there is a key somewhere. I could look through the code but for now it seems it "just works".

I wouldn't call the customization as "poorly understood" but if we can stick with the default, all the better

Reply:
> Apparently ccache-action enumerates caches, so it can pick one and resstore it; I suppose there is a key somewhere.

Yes, we define that key (there is even another change request about that definition).

 
> I wouldn't call the customization as "poorly understood" 

I do not understand it. Documentation says pretty much nothing. Looking through code does not offer enough clues (for me). Cache entry creation time (or equivalent) is probably (or should be) available without appending that time to the entry name or necessitating adding a configuration option about it. Your understanding has not been enough to explain why this customization is needed (or how it works) to me. Nobody else is participating in developing this understanding. I think all these factors, collectively, qualify as "poorly understood" :-)!

> if we can stick with the default, all the better

Exactly!